annotations/views.py

```diff
@@ -218,6 +218,44 @@ def list_texts(request):
     }
     return HttpResponse(template.render(context))
 
+@login_required
+def list_user(request):
+    """
+    List all the users of Vogon web
+    """
+
+    template = loader.get_template('annotations/contributors.html')
+
+    #Default Sort
+    sort = "user_name"
+    #To map the column sort parameter from UI to the column name in DB
+    sort_dict = {"user_name":"username", "name":"full_name",
+     "aff":"affiliation", "loc":"location"}
+
+    if(request.GET.get('sort') != None):
```

Comment:
A simpler way to do this would be to provide the alternate (default) value to ``GET.get()``, e.g.:

```python
sort_column = request.GET.get('sort', 'user_name')    # If no 'sort' key, then returns ``'user_name'``
```